app-modules/ai/resources/views/components/assistant.blade.php

```diff
@@ -536,8 +536,19 @@ class="col-span-1 flex flex-col gap-2 overflow-hidden pt-3 lg:col-span-2 lg:pt-6
                     @endif
                 </div>
 
+                @php
+                    $isInstitutionalAdvisor = $this->thread->assistant->name === 'Institutional Advisor';
```

Comment:
Instead of checking for the name here you should check if the assistant `is_default`. It may not always have the same name but what you are trying to do here is see if it is the default advisor.